spec/cli_helper.rb

```diff
@@ -14,3 +14,9 @@ def capture_stdio(input = nil, &block)
   alias capture_stdout capture_stdio
 end
 
+RSpec.configure do |config|
+  config.after(:each) do
+    # To imitate the initial class load before calling `VimGolf::CLI.start`
+    VimGolf.ui = VimGolf::UI.new
```

Comment:
Hmm, isn't this already handled by https://github.com/igrigorik/vimgolf/blob/master/lib/vimgolf/lib/vimgolf/cli.rb#L31? I think I'm missing something obvious here. :)